README.md

```diff
@@ -156,6 +157,10 @@ A curated list of awesome things related to <a href='https://github.com/vitejs/v
 - [vite-plugin-importer](https://github.com/ajuner/vite-plugin-importer) - Integration for babel-plugin-import.
 - [vite-plugin-banner](https://github.com/chengpeiquan/vite-plugin-banner) - Adds a banner to the top of each generated chunk.
 
+#### Dev Servers
+
+- [vite-plugin-node](https://github.com/axe-me/vite-plugin-node) - Integration with Node.js backend servers.
```

Comment:
Let's move it to the #Integrations section for now. We will see if need to separate this out when we have enough plugins in this category.

Reply:
moved, also fixed the duplicated bundling sections